management/src/clusterm/manager/client_test.go

```diff
@@ -121,16 +150,16 @@ func (s *managerSuite) TestPostSuccess(c *C) {
 			nodeName:  testNodeName,
 			extraVars: "",
 			hostGroup: "",
-			exptdBody: reqCommissionWithoutHostGroupBody.Bytes(),
+			exptdBody: reqEmptyBody.Bytes(),
 			cb:        clstrC.PostNodeCommission,
 		},
 		"commission-extra-vars": {
-			expURLStr: fmt.Sprintf("http://%s/%s/%s?%s=%s",
-				baseURL, PostNodeCommissionPrefix, testNodeName, ExtraVarsQuery, testExtraVars),
+			expURLStr: fmt.Sprintf("http://%s/%s/%s",
```

Comment:
minor: this is too short a line we can remove the line break. Same applies to most cases below as well.